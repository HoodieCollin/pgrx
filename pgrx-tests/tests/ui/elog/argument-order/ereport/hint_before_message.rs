use pgrx::prelude::*;

fn main() {}

#[pg_extern]
fn ereport_hint_before_message_is_invalid() {
    ereport!(
        loglevel = PgLogLevel::LOG;
        errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
        hint = "hint";
        message = "...";
    );
}
