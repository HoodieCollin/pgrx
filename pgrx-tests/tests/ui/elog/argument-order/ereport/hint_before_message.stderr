error: The alternative invocation style requires `message` argument to be specified after `errcode`.
  --> tests/ui/elog/argument-order/ereport/hint_before_message.rs:7:5
   |
7  | /     ereport!(
8  | |         loglevel = PgLogLevel::LOG;
9  | |         errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
10 | |         hint = "hint";
11 | |         message = "...";
12 | |     );
   | |_____^
   |
   = note: this error originates in the macro `ereport` (in Nightly builds, run with -Z macro-backtrace for more info)
